nautobot/dcim/models/racks.py

```diff
@@ -391,9 +391,9 @@ def get_elevation_svg(
             Defaults to True, showing device full name and hiding truncated.
         """
         if unit_width is None:
-            unit_width = get_settings_or_config("RACK_ELEVATION_DEFAULT_UNIT_WIDTH")
+            unit_width = get_settings_or_config("RACK_ELEVATION_DEFAULT_UNIT_WIDTH", fallback=230)
         if unit_height is None:
-            unit_height = get_settings_or_config("RACK_ELEVATION_DEFAULT_UNIT_HEIGHT")
+            unit_height = get_settings_or_config("RACK_ELEVATION_DEFAULT_UNIT_HEIGHT", fallback=22)
```

Comment:
Should we make these two numbers into a settings variables like `RACK_ELEVATION_DEFAULT_FALLBACK_UNIT_HEIGHT`?

Reply:
Not settings variables, but could make them constants. The fallback is *only* used if:
- not defined in settings/nautobot_config.py
- unable to read the Constance database entry (due to migrations not yet run, etc.)
so it would be redundant to make the fallback also user-configurable.